js/index.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable camelcase */
+import { SlideDeck } from './slidedeck.js';
+
+const map = L.map('map', { scrollWheelZoom: false }).setView([0, 0], 0);
+const mapboxKey = 'pk.eyJ1IjoiY2xhdWRsb3ciLCJhIjoiY20weTY3MDZoMDNocTJrbXpqa3lqZWJlaSJ9.3N1iXpEvsJ0GwajGVwwkTg';
+const mapboxStyle1 = 'mapbox/light-v11';
+
+// ## The Base Tile Layer
+const baseTileLayer = L.tileLayer(`https://api.mapbox.com/styles/v1/${mapboxStyle1}/tiles/512/{z}/{x}/{y}{r}?access_token=${mapboxKey}`, {
+  maxZoom: 16,
+  attribution: '&copy; <a href="https://mapbox.com/" target="_blank">Mapbox</a> &copy; <a href="https://stamen.com/" target="_blank">Stamen Design</a> &copy; <a href="https://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a>',
+});
+baseTileLayer.addTo(map);
+
+// ## The SlideDeck object
+// ## Interface Elements
+const slides = document.querySelectorAll('.slide');
+
+// ## The SlideDeck object
+// ## Icons and Styling Options
+const fridgeIcon = L.icon({
+  iconUrl: 'data/fridgeicon.png',
+  iconSize: [20, 20],
+  iconAnchor: [15, 15],
+  popupAnchor: [0, -15],
+});
+
+const shelterIcon = L.icon({
+  iconUrl: 'data/sheltericon.png',
+  iconSize: [20, 20],
+  iconAnchor: [15, 15],
+  popupAnchor: [0, -15],
+});
```

Comment:
**suggestion:** I might recommend making these icons larger, so that the locations stand out a little more against the map. Since you already have your anchors at `[15, 15]`, making the icons 30px would make sense anyway (for a circular marker, the anchor should usually be in the center of the dot).
```suggestion
const fridgeIcon = L.icon({
  iconUrl: 'data/fridgeicon.png',
  iconSize: [30, 30],
  iconAnchor: [15, 15],
  popupAnchor: [0, -15],
});

const shelterIcon = L.icon({
  iconUrl: 'data/sheltericon.png',
  iconSize: [30, 30],
  iconAnchor: [15, 15],
  popupAnchor: [0, -15],
});
```